.pre-commit-config.yaml

```diff
@@ -1,13 +1,13 @@
--   repo: https://gitlab.com/pycqa/flake8
-    rev: ''
+-   repo: https://github.com/ambv/black
+    rev: 19.10b0
```

Comment:
so when a stable version of this release is out do we plan to bump it? (I know the versioning isn't changing in this PR, but I'm just curious!).

Reply:
I guess we should at some point. Although that could again have large scale effects on the code base, since Black devs state the style _might_ change in the future (not in any major way though).

Reply:
Yeah, I think when there's a stable release we should bump to that, since I don't think it's that good if we have multiple beta releases (or bleeding edge packages) needed to do dev work on yt. But we can deal with that (and the cascading effects) later. 🙂

Reply:
I agree !

Reply:
In the mean time I'm watching all repos of tools related to YTEP0037, I get notifications when they do releases :)